node_modules/gh-orgunit/lib/rest.js

```diff
@@ -458,3 +458,125 @@ var unsubscribeOrgUnit = function(req, res) {
 
 GrassHopper.globalAdminRouter.on('post', '/api/orgunit/:id/unsubscribe', unsubscribeOrgUnit);
 GrassHopper.appRouter.on('post', '/api/orgunit/:id/unsubscribe', unsubscribeOrgUnit);
+
+/**
+ * @REST exportOrgUnit
+ *
+ * Export the organisational units, event series and events under an organisational unit
+ *
+ * The data can be exported in JSON or CSV. The JSON export format is fairly straightforward and
+ * contains nested organisational units. The CSV format is not entirely non-trivial and requires
```

Comment:
There is some language that needs fixing up in this documentation (e.g. not entirely non-trivial, is a flat structure it's a bit awkward to support, etc.)